packages/fxa-content-server/app/styles/tailwind/ctas.css

```diff
@@ -3,10 +3,6 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 /* Max-height is a likely temp "hack" for .spinner until it's converted to TW */
```

Comment:
@millsoper This max-height comment still needs to be moved over, buuuut Valerie ended up needing this class in #14599 which is about to be merged so you will need to fix conflicts but they should be easy. 😅

Reply:
Ohhh I thought this was separate since it was referring to a spinner 😬 I'll move it over